docs/upgrade.md

```diff
@@ -6,202 +6,133 @@ sidebar_label: Upgrade
-wget https://raw.githubusercontent.com/kmova/openebs/599e062a2251d2c16bca59aeac4a7ed77e445e6e/k8s/upgrades/0.5.x-0.6.0/replica.patch.tpl.yml
-```
-
-In 0.5.x releases, when a replica is shutdown, it gets rescheduled to another available node in the cluster and starts copying the data from other replicas. This is not a desired behaviour during upgrades as it creates new replicas due to rolling-upgrade. To pin the replicas or force them to the nodes where the data is already present, starting with 0.6, OpenEBS uses the concept of NodeSelector and Tolerations that ensures replicas are not moved on node while deleting pod.
+Limitations:
+- This is a preliminary script only intended for using on volumes where data has been backed up.
+- Please have the following link handy in case the volume gets into read-only during upgrade. 
+  https://docs.openebs.io/docs/next/readonlyvolumes.html
+- Automatic rollback option is not provided. To rollback, you must update the controller, exporter, and replica pod images to the   previous version.
+- In the process of running the following procedure, if you run into issues, you can always reach us on Slack.
 
-As part of upgrade, OpenEBS recommends that you label the nodes where the replica pods are scheduled as follows.
 
 ```
-kubectl label nodes gke-kmova-helm-default-pool-d8b227cc-6wqr "openebs-pv"="openebs-storage"
+kubectl get pv
 ```
 
-Note that the key `openebs-pv` is fixed. However you can use any value in place of `openebs-storage`. This value will be taken as a parameter in the upgrade script below. 
-
-Repeat the above step of labellilng the node for all nodes where replicas are scheduled. Assumption is that all the PV replicas are scheduled on the same set of 3 nodes. 
-
-
-**Limitations:**
-
-- Must handle cases where there are a mix of PVs with 1 and 3 replicas or 
-- Scenarios like PV1 replicas are on nodes n1, n2, and n3, whereas PV2 replicas are on nodes n2, n3, and n4.
-- This is a preliminary script only intended for use on volumes where data is backed-up.
-- Have the following link handy in case the volume gets into read-only state during the upgrade process.
-  [readonlyvolumes.html](https://docs.openebs.io/docs/next/readonlyvolumes.html)
-- In the process of running the following steps, if you run into issues, you can always reach us on Slack.
-
-1. Select the PV that must be upgraded.
-
-2. Use the following command to get the PV.
-
 ```
-kubectl get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                          STORAGECLASS      REASON    AGE
+pvc-48fb36a2-947f-11e8-b1f3-42010a800004   5G         RWO            Delete           Bound     percona-test/demo-vol1-claim   openebs-percona             8m
 ```
 
-3. Upgrade the PV that you want to upgrade by using the following command.
+### Upgrade the PV that must be Upgraded 
 
```

Comment:
Heading = Upgrade the PV
Para = Upgrade the PV that you want to upgrade using the following command.